framework/util/file_path.cpp

```diff
@@ -435,6 +435,30 @@ void GetApplicationInfo(FileInfo& file_info)
     {
         GFXRECON_LOG_WARNING("Failed to retrieve the application executable name");
     }
+#elif defined(__ANDROID__) || defined(__linux__)
+    std::ifstream cmdlineFile("/proc/self/cmdline");
+
+    if (cmdlineFile.is_open())
+    {
+        std::string line;
+        std::getline(cmdlineFile, line);
+
+        size_t pos = std::string::npos;
+        // Command line arguments separated by null bytes
+        if ((pos = line.find('\0')) != std::string::npos)
+        {
+            strcpy(file_info.AppName, line.substr(0, pos).c_str());
```

Comment:
I'd prefer strncpy here

Reply:
Sorry, my mistake, @davidd-lunarg points out that we have a platform `StringCopy` function, please check that out.

Reply:
Done!